post_processing/ci_plots.jl

```diff
@@ -542,19 +543,112 @@ EDMFBoxPlots = Union{
 function make_plots(::EDMFBoxPlots, simulation_path)
     simdir = SimDir(simulation_path)
 
-    short_names = ["ua", "wa", "thetaa", "taup", "haup", "waup", "tke", "arup"]
+    short_names = [
+        "ua",
+        "wa",
+        "thetaa",
+        "thetaaup",
+        "ta",
+        "taup",
+        "ha",
+        "haup",
+        "waup",
+        "tke",
+        "arup",
+        "hus",
+        "husup",
+        "hur",
+        "hurup",
+        "cl",
+        "clw",
+        "cli",
+    ]
     reduction = "average"
     period = "10m"
     vars = [
         get(simdir; short_name, reduction, period) for short_name in short_names
     ]
     vars_zt = [slice(var, x = 0.0, y = 0.0) for var in vars]
     vars_z = [slice(var, x = 0.0, y = 0.0, time = LAST_SNAP) for var in vars]
-    make_plots_generic(
-        simulation_path,
-        [vars_zt..., vars_z...],
-        more_kwargs = YLOGSCALE,
-    )
+
+    summary_files = make_plots_edmf_vars_z(simulation_path, vars_z)
+    make_plots_generic(simulation_path, vars_zt, summary_files = summary_files)
+end
+
+long_name(var) = var.attributes["long_name"]
+short_name(var) = var.attributes["short_name"]
+
+function parse_long_name(var)
+    name = long_name(var)
+    for str in ["east ", "west ", "north ", "south "]
+        name = replace(name, "degrees_" * str => "")
+    end
+    var_name, rest = split(name, ",")
+    reduction, rest = split(rest, "(s) ")
+    coord, time = split(rest, " time = ")
+    return "$(short_name(var)), $coord, t = $time"
+end
```

Comment:
What is your goal here?

Reply:
Printing the full long name is much too long for a grid of plots, so this keeps the necessary info and makes it shorter

Reply:
What do you want to keep? This is function is not very robust. We try to add some extra information in the attributes of OutputVar.

Reply:
Yes, this function is pretty ugly. It would be nice to have a field for coordinate and timestamp when it is applicable.